### ISSUE TITLE
Allow Org Admins Add Users to their Orgs

### ISSUE THREAD

**Issue by mattttt** [raintank/grafana#319]
Currently, admin users within an org are not able to add users to their organization. Chatted with @nopzor1200 about this last night, and we couldnt recall any non-technical reasons why we were preventing this. 

Please adjust to let Org Admins add Users, but retain existing functionality that prevents Org Admins from creating new Orgs.

**Comment by torkelo** [comment]
Org Admins can add Org users (Org user = link between user account and org). There is currently no screen/view where an org admin can create user accounts, is that what you mean?

**Comment by mattttt** [comment]
In essence, yes. An Org Admin can currently associate a user to an Org, but it requires a Grafana Admin to add that user - which breaks if a new Litmus user wanted to add a teammate.

**Comment by woodsaj** [comment]
This was a deliberate choice to control the number of users we have.

If we want to change this, then we can just  enable user signups.  Once the user has an account an Org Admin can then grant them access to the Org.

**Comment by mattttt** [comment]
I see what some of the confusion may be. We dont want to allow all user signups, nor do we want them to be able to have their own org (before being added to the company). 

We need to find a way to have a new user added, but only associated with the Org in which it was created within.

**Comment by nopzor1200** [comment]
So in thinking about this, the main problem I had with allowing Org admins to add users for the beta was that for untrusted Org admins, adding Org users is indeed a backdoor while there are no quotas in place for endpoints and collectors.

Eg. Org admin gives users to 1000 friends and they collectively add hundreds or thousands of endpoints using the Org as a shared account of sorts.

It is important to allow Litmus Org admins the ability to invite their coworkers. But I think we can't do it until there are some quotas in place.

I'll open a separate issue for quotas. It is definitely shaping up to be a high priority item as we start opening the beta up.

**Comment by nopzor1200** [comment]
Once quotas #321 is in place we can allow Orgs to add users to their Org in Litmus. The way it will happen will likely be related to the ongoing Grafana user invite system that @torkelo is prioritizing.

**Comment by nopzor1200** [comment]
Quotas are now in place per @woodsaj but we have not deployed the codebase to prod because we've encountered some (possibly related) strange behavior.

**Comment by woodsaj** [comment]
Quotas are deployed in Production.

@mattttt   are we just now waiting on https://github.com/grafana/grafana/issues/2353 ?

**Comment by torkelo** [comment]
https://github.com/grafana/grafana/issues/2353 is pretty much done, waiting for feedback on the final signup flow. 

But the first part (org admins can now invite new users) seems to be in production already

**Comment by woodsaj** [comment]
This was implemented and is already live in production.
